flowlog support

giggsoff opened the pull request:
Support flowlog command in Eden to show network flow statistics (TCP and UDP flows with IP addresses, port numbers, counters, whether dropped or accepted),

Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>

mydatascience commented:
While this is definitely good, it will be great to have some sample output and more description.
Also why is this called flowlog? Not netstat or something like that. 
Where is it used?

giggsoff commented:
> While this is definitely good, it will be great to have some sample output and more description.

File with examples added

> Also why is this called flowlog? Not netstat or something like that.

Renamed to netstat

> Where is it used?

Added to test
